doc/r_interface.md

```diff
@@ -3,16 +3,18 @@
 
 ####  Install Rcpp package  
  
-* Install package-dependencies: `Rcpp`, `RcppEigen`, `BH`, `lpSolveAPI`.  
+* Install package-dependencies: `Rcpp`, `RcppEigen`, `BH`.  
 
-1. Then use devtools package to install `volesti` Rcpp package. In folder /root/R-proj Run:
+1. Then use devtools package to install `volesti` Rcpp package. From terminal go to folder /root/R-proj and run in `R`:  
```

Comment:
why not just `Rscript -e 'Rcpp::compileAttributes()'`

Reply:
without entering and exiting `R`

Reply:
Thank you @vissarion! Done.